rs/nns/governance/src/governance.rs

```diff
@@ -3278,6 +3278,19 @@ impl Governance {
         .map_err(GovernanceError::from)
     }
 
+    fn set_following(
+        &mut self,
+        id: &NeuronId,
+        caller: &PrincipalId,
+        set_following: &manage_neuron::SetFollowing,
+    ) -> Result<(), GovernanceError> {
+        for request in set_following.clone().into_vec_of_follow() {
+            self.follow(id, caller, &request)?;
```

Comment:
```suggestion
            self.follow(id, caller, &request)?;
```

I wonder if the errors should be remapped to avoid confusing the user in some cases. For example, does the error say which command failed? If so, it should be set-following, not follow, which would fail. Right?

Reply:
I looked at all the places where follow can return, and it seems like all the errors (also) are suitable for set_following.